testsuite/MDAnalysisTests/analysis/test_dssp.py

```diff
@@ -32,8 +69,6 @@ def test_trajectory(client_DSSP):
     assert (
         first_frame[:10] != last_frame[:10] == avg_frame[:10] == "-EEEEEE---"
     )
-    protein = mda.Universe(TPR, XTC).select_atoms("protein")
-    run = DSSP(protein).run(**client_DSSP, stop=10)
```

Comment:
These lines seemed superfluous as the atomgroup approach is tested separately.

Reply:
agree